cli/lib/kontena/cli/stacks/yaml/validator_v3.rb

```diff
@@ -69,6 +69,9 @@ def validate(yaml)
         if yaml.key?('services')
           if yaml['services'].kind_of?(Hash)
             yaml['services'].each do |service, options|
+              unless service =~ /\A[a-z0-9\-]+\z/
```

Comment:
Is there any reason for the CLI and the server `GridServices::Create` to use a different regexp for validation?

https://github.com/kontena/kontena/blob/6f9de9d5518b3e3d72516906fbf25d4151b18fcf/server/app/mutations/grid_services/create.rb#L15

Reply:
`\w includes underscore`. But maybe should add `(?!-)` here